goal_src/jak2/engine/target/logic-target.gc

```diff
@@ -3384,16 +3385,17 @@
   (logior! (-> obj skel status) (joint-control-status sync-math blend-shape eye-anim))
   (set! (-> obj draw shadow-ctrl) *target-shadow-control*)
   (set! (-> obj shadow-backup) (-> obj draw shadow))
-  (let ((t9-24 (method-of-type carry-info new))
-        (a0-65 'process)
-        (a1-23 carry-info)
+  (set! (-> obj carry)
+        (new
+          'process
+          'carry-info
+          obj
+          41
+          (the-as vector (new 'static 'array float 1 0.0))
+          (the-as vector (new 'static 'array float 1 0.0))
```

Comment:
This is probably going to cause a bug. The label types in this file for these values should probably be `vector`, not an array of 1 float. As written, the `new` method will get junk if it tries to access y, z, or w from this vector.